.gitattributes

```diff
@@ -1,2 +1,3 @@
 * text=auto
 scripts/* text eol=lf
+docker/minio/* text eol=lf
```

Comment:
Maybe at this point, it'd be easier to just declare all `*.sh` files with LF line endings

Reply:
yeah